astropy/nddata/tests/test_ccddata.py

```diff
@@ -643,6 +643,9 @@ def test_wcs_attribute(tmpdir):
     assert ccd_new_hdu_mod_wcs.header['CDELT2'] == ccd_new.wcs.wcs.cdelt[1]
 
 
+@pytest.mark.filterwarnings(
```

Comment:
I think we should capture and verify this warning inside the test - because the reason for the warning is explicitly tested here.

Reply:
Okay, done! Thanks.